### PR TITLE
Added re-frisk debugger

### PR DESCRIPTION
**Pull request by flexsurfer**
[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

### Summary:
[comment]: # (Summarise the problem and how the pull request solves it)
image

### Steps to test:
- run "lein re-frisk use-re-natal" after running re-natal commands 
- run dev status 
- open http://localhost:4567/ in the browser

[comment]: # (PRs will only be accepted if squashed into single commit.)
status: ready

### CODE REVIEW

**Comment by alwx** [comment]
Think we should merge it because it makes the development process easier

File: src/status_im/android/core.cljs [hunk 1]
```diff
@@ -135,6 +136,7 @@
                       [component])]])]))))})))
 
 (defn init []
+  (dev/dev-setup)
```
Comment (rasom) [inline src/status_im/android/core.cljs]: @flexsurfer, let's move this line to `env.android.main` and `env.ios.main`. I suppose we will use `re-frisk` only for development, so no need to add it to `status-im.*.core` namespaces

**Comment by rasom** [comment]
screenshot_2017-03-12-21-10-59
^ I have this error when i'm trying to run app on real android device, presumably because i have to `adb reverse` some port to make it work or something...

Also it's not good that app crashes when a connection to some development tool (which is obviously not required)  fails, I would prefer to see warnings there.

Another thing.. @flexsurfer, it would be great if you can provide some API (or maybe it already exists) to make it possible to start `re-frisk` from our `user` ns, in this case we will not need to run `lein re-frisk` separately (as it works with [`figweel`)](https://github.com/status-im/status-react/blob/develop/env/dev/user.clj#L32).

File: src/status_im/android/core.cljs (hunk 1 above)
Comment (flexsurfer) [inline src/status_im/android/core.cljs]: sounds reasonable thanks , i'll do it

**Comment by flexsurfer** [comment]
@rasom you're right, tool shouldn't crash the app, i've fixed it, yes i have an API , i'll implement it in the user ns

**Comment by flexsurfer** [comment]
@rasom done, debugger works fine on ios simulator, but doesn't work in genymotion, though application works without errors. We need to find a way to replace host and port for genymotion and real devices like it works for figwheel

**Comment by rasom** [comment]
`re-natal` just replaces websocket's url in `env.android/ios.main` by regexp https://github.com/drapanjanas/re-natal/blob/4b2f82d2a4cf11f3646863a963b6cd99cc327b2a/re-natal.coffee#L30. so we could do the same

**Comment by flexsurfer** [comment]
@rasom  implemented "lein re-frisk use-re-natal"

**Pull request merged**